imap/jmap_sieve.c

```diff
@@ -127,6 +127,16 @@ static jmap_method_t jmap_sieve_methods_nonstandard[] = {
 
 HIDDEN void jmap_sieve_init(jmap_settings_t *settings)
 {
+    if (config_getswitch(IMAPOPT_SIEVEUSEHOMEDIR)) {
+        xsyslog(LOG_NOTICE, "%s", "can't use home directories");
+        return;
+    }
+
+    if (!sievedir_valid_path(config_getstring(IMAPOPT_SIEVEDIR))) {
+        xsyslog(LOG_NOTICE, "%s", "sievedir option is not defined/valid");
```

Comment:
Isn't that rather an error than just a notice? Same for the check of sieveusehomedir.

Reply:
I guess it's the "valid" part in "not defined/valid" that triggered me. If it's invalid, it is more likely and error, but I don't know really.